driver-check.sh

```diff
@@ -10,6 +10,22 @@ warnings=0
 trap 'rm -rf "$tmp"' EXIT
 tmp=$(mktemp -d)
 
+find_usrmerge_boot() {
+    local filename=$1
+    local kver=$2
+    local ext=${3:+."$3"}
+    local f
+
+    for f in "/boot/$filename-$kver$ext" "/usr/lib/modules/$kver/$filename$ext"
```

Comment:
maybe reverse the order to always prefer the real location? /boot might be compat symlink only

Reply:
no is ok, that's why I didn't flag it as "request changes". Still feel like it's slightly more correct to revere the order, even though academic in practice :-)

Reply:
ok, I changed it.